File: Prak605/PRAK605-2210817210021-TRISNACAHYAPERMADI.c
#include <stdio.h>
int main(void){
    int m1[10][10], m2[10][10], hasil[10][10];
    int a, b, x, y, jumlah = 0;
    scanf ("%i", &y);
    printf ("Matriks A\n");
    for (a = 0; a < y; a++){
        for (b = 0; b < y; b++){
            scanf ("%i", &m1[a][b]);
        }}
    printf ("Matriks B\n");
    for (a = 0; a < y; a++){
        for (b = 0; b < y; b++){
            scanf ("%i", &m2[a][b]);
        }}
    for (a = 0; a < y; a++){
        for (b = 0; b < y; b++){
            for (x = 0; x < y; x++){
                jumlah = jumlah + m1[a][x] * m2[x][b];}
                hasil[a][b] = jumlah;
                jumlah = 0;}}
                printf ("Matriks AXB\n");
                for (a = 0; a < y; a++){
                    for (b = 0; b < y; b++){
                        printf(" %i ", hasil[a][b]);}
                        printf("\n");}               
}

File: Prak605/PRAK605-2210817210021-TRISNACAHYAPERMADI.py
x = int(input())
matriks1 = []; matriks2 = []; matriksAB = []
print ("Matriks A")
for i in range (x):
    y = list(map(int, input().split()))
    matriks1.append(y)
print ("Matriks B")
for i in range (x):
    y = list(map(int, input().split()))
    matriks2.append(y)
print ("Matriks AXB")
for i in range (x):
    matriksAB.append([])
    for j in range (x):
        a = 0
        for y in range (x):
            a = a + matriks1[i][y] * matriks2[y][j]
        matriksAB[i].append(a)
for input in matriksAB:
    for y in input:
        print(y, end=" ")
    print()
